genre-recommendation/graph_sage_classification_task.sql

```diff
@@ -0,0 +1,79 @@
+USE ROLE ACCOUNTADMIN;
+
+-- Let's assume you have a database named 'genre_classification_db' which contains data in the schema 'imdb'
+-- and 'results' for storing the results of algorithm predictions.
+-- Application name is Neo4j_GDS
+GRANT USAGE ON DATABASE genre_classification_db TO APPLICATION Neo4j_GDS;
+GRANT USAGE ON SCHEMA genre_classification_db.results TO APPLICATION Neo4j_GDS;
+GRANT USAGE ON SCHEMA genre_classification_db.imdb TO APPLICATION Neo4j_GDS;
+
+GRANT CREATE STAGE ON SCHEMA genre_classification_db.results TO APPLICATION Neo4j_GDS;
+GRANT CREATE STAGE ON SCHEMA genre_classification_db.imdb TO APPLICATION Neo4j_GDS;
```

Comment:
which stages is the application creating here?

Reply:
Removed